app/schema.graphql

```diff
@@ -1220,7 +1220,7 @@ type Project implements Node {
   spanLatencyMsQuantile(probability: Float!, timeRange: TimeRange, filterCondition: String): Float
   trace(traceId: ID!): Trace
   spans(timeRange: TimeRange, first: Int = 50, last: Int, after: String, before: String, sort: SpanSort, rootSpansOnly: Boolean, filterCondition: String): SpanConnection!
-  sessions(timeRange: TimeRange, first: Int = 50, after: String): ProjectSessionConnection!
+  sessions(timeRange: TimeRange, first: Int = 50, after: String, substring: String): ProjectSessionConnection!
```

Comment:
same nit here: filter is probably overloaded but substring feels maybe a bit too vague maybe searchString, or just search since string will be implied by the type. what do you think?

Reply:
ok will rename an filter_substring